transform/transformers/PDFTransformer.py

```diff
@@ -99,6 +100,9 @@ def get_table_data(self, question_group):
                 answer = ''
                 if question['question_id'] in self.response['data']:
                     answer = self.response['data'][question['question_id']]
-                table_data.append([Paragraph(question['text'], styleN), Paragraph(answer, styleN)])
+
+                wrapped_answer = "\n".join(wrap(answer, 35))
```

Comment:
It would be better if this number was defined as a constant e.g. MAX_ANSWER_CHARACTERS_PER_LINE